### ISSUE TITLE
Replace "couple" with "tuple" in definition of DataLoader in 06_multicat

### ISSUE THREAD

**Issue by mccallion** [fastai/fastbook#229]
"Couple" is grammatically correct, but I believe "tuple" was intended because each mini-batch is a tuple of a couple of x's and y's.

From the nb:

### 1.1.4  Constructing a DataBlock
How do we convert from a `DataFrame` object to a `DataLoaders` object? We generally suggest using the data block API for creating a `DataLoaders` object, where possible, since it provides a good mix of flexibility and simplicity. Here we will show you the steps that we take to use the data blocks API to construct a `DataLoaders` object in practice, using this dataset as an example.

As we have seen, PyTorch and fastai have two main classes for representing and accessing a training set or validation set:

`Dataset`:: A collection that returns a tuple of your independent and dependent variable for a single item
`DataLoader`:: An iterator that provides a stream of mini-batches, where each mini-batch is a **couple** of a batch of independent variables and a batch of dependent variables

**Comment by TannerGilbert** [comment]
@mccallion maybe close this issue as a pull request has already been merged.
